### PR TITLE
Updates deps and fixes tests for CommonJS and ESModules.

### LINKED ISSUE #30: commonjs support?
I know it's 2023 and all ...

### PR DESCRIPTION
**Pull request by armfazh**
Changes:
- Tests must run for both ESM and CJS.
- Updates and ping dependencies.
- known bug: Jest doesn't recognize collectCoverage and verbose from configuration. 
- Fix bench compilation

Fixes #30

### CODE REVIEW

**Review by thibmeu — COMMENTED** [review]
change looks good, apart from `coverage` which does not work and prints warnings

File: bench/group.bench.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export async function benchGroup(bs: Benchmark.Suite) {
     const dst = te.encode('dst')
 
     for (const id of Object.values(Oprf.Group.ID)) {
-        const gg = new Oprf.Group(id)
+        const gg = Oprf.Group.fromID(id)
```
Comment (thibmeu) [inline bench/group.bench.ts:25]: side comment: that's a good modification.

File: jest.config.mjs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        transform: {},
+        setupFiles: [`./lib/${version}/mockCrypto/mock_crypto.js`],
+        collectCoverage: true,
+        coverageDirectory: `coverage/${version}`,
+        verbose: true
+    }
+}
+
+export default {
+    projects: [getConfig('esm'), getConfig('cjs')]
```
[the remark this answers is not in the document]
Comment (armfazh) [reply jest.config.mjs:21]: main issue of this approach: relies on the use of ENV variables, and only generates the code for one variant.

[the remark this answers is not in the document]
Comment (armfazh) [reply jest.config.mjs:21]: after RTFM, there is an easy way to pass the options.

[the remark this answers is not in the document]
Comment (armfazh) [reply jest.config.mjs:20]: well, I went with the verbose approach, passing args to the CLI

**Review by thibmeu — APPROVED** [review]

**Pull request merged**